test_ambaridb-onefs.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
+
+# Usage info
+show_help() {
+cat << EOF
+Usage: ${0##*/} [-o ONEFSHOST]
+Execute on host with Ambari Server to correct the host and cluster state due to
+bug in OneFS 8.0.0.3 and 8.0.1.0 when used with Ambari 2.4.x.
+
+    -o ONEFSHOST   The OneFS host name as it appears in Ambari Server.
+EOF
+}
+
+ONEFSHOSTNAME=""
+
+OPTIND=1
+
+while getopts ho: opt; do
+    case $opt in
+        h)
+            show_help
+            exit 0
+            ;;
+        o)  ONEFSHOSTNAME=$OPTARG
```

Comment:
I don't think this script will function if this isn't provided. Perhaps we should make it an arg instead of an opt and detect if it isn't passed so we can print a nice error message and bail.